libs/types/src/cdf/cast-vote-records/vx-schema.json

```diff
@@ -1214,7 +1222,31 @@
           "type": "string"
         },
         "BatchLabel": {
+          "description": "A human readable label for the batch.",
           "type": "string"
+        },
+        "SequenceId": {
+          "description": "The ordinal number of the batch in the tabulator's sequence of batches in a given election.",
+          "type": "integer"
+        },
+        "StartTime": {
+          "description": "The start time of the batch. On a precinct scanner, the start time is when the polls are opened or voting is resumed. On a central scanner, the start time is when the user initiates scanning a batch.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "EndTime": {
+          "description": "The end time of the batch. On a precinct scanner, the end time is when the polls are closed or voting is paused. On a central scanner, the end time is when a batch scan is complete",
+          "type": "string",
+          "format": "date-time"
+        },
+        "Size": {
```

Comment:
maybe `NumberSheets` would be more CDF-y (thinking of `NumberVotes`)

Reply:
Great!